ui/helpers/utils/display-critical-error.ts

```diff
@@ -24,26 +24,51 @@ export enum CriticalErrorTranslationKey {
  * @throws {ErrorLike} Throws the error after displaying the message.
  * @returns A promise that resolves to never, as it always throws an error.
  */
-export async function displayCriticalError(
+export async function displayCriticalErrorMessage(
   container: HTMLElement,
   errorKey: CriticalErrorTranslationKey,
   error: ErrorLike,
   currentLocale?: string,
-): Promise<never> {
+) {
```

Comment:
I didn't mean to remove this
```suggestion
): Promise<never> {
```